azure/requests.yaml

```diff
@@ -34,3 +24,7 @@ releases:
           issue: https://github.com/giantswarm/giantswarm/issues/13671
         - name: kubernetes
```

Comment:
@whites11 you wanted 1.17.12 for the alpha release right? is this request covering alpha as well? For >=13.0.0 we want 1.18.x. And if it covers alphas then this PR would create a mess..